app/views/registrations/edit_registrations.html.erb

```diff
@@ -1,5 +1,5 @@
 <% provide(:title, I18n.t('registrations.list.title', comp: @competition.name)) %>
 
 <%= render layout: 'nav' do %>
-  <%= react_component('RegistrationsV2/RegistrationAdministration', { competitionInfo: @competition.to_competition_info }) %>
+  <%= react_component('RegistrationsV2/RegistrationAdministration', { competitionId: @competition.id }) %>
```

Comment:
Why is this change in approach necessary? And will it perhaps be "fixed" by the Registration-Context PR?

Reply:
(to discuss in meeting_

Reply:
- [ ] Implement a route that returns the auto_accept_enabled state